include/TRFFitter.h

```diff
@@ -3,8 +3,8 @@
 
 #include <vector>
 
-#include "TNamed.h"
-#include "Rtypes.h"
+#include <TNamed.h>
+#include <Rtypes.h>
```

Comment:
Nope. But that's a good idea. Maybe I'll write up something quick and dirty for the wiki sometime this week. It would be handy!